libs/admin/src/lib/state/admin.effects.ts

```diff
@@ -76,7 +76,9 @@ export class AdminEffects {
     this.actions$.pipe(
       ofType(actions.deleteWorkgroup),
       switchMap(({ workgroupId }) =>
-        this.adminService.deleteWorkgroup(workgroupId).pipe(map(() => actions.listWorkgroups()))
+        this.adminService
+          .deleteWorkgroup(workgroupId)
+          .pipe(map(() => actions.removeWorkgroupAfterDelete({ workgroupId })))
       )
```

Comment:
For readability, I pretty much always prefer to move `pipe` out of `switchMap`:
```suggestion
      switchMap(({ workgroupId }) => this.adminService.deleteWorkgroup(workgroupId)),
      map(() => actions.removeWorkgroupAfterDelete({ workgroupId })),
```

Reply:
Outside the switchamp i no longer have access to the `workgroupId`. I would have to return it fro within the adminService, which seems unnecessary to me

Reply:
very true, did not see that